### ISSUE TITLE
Preconfigured data sources of type GDAL not working

### ISSUE THREAD

**Issue by sfkeller** [nextgis/quickmapservices#77]
I don't see any layer content in QGIS nor an error when choosing following two services of type GDAL:
1. "eatlas EGIP": from src/data_sources/eatlas_egip/gdal_source.xml
2. Rosreestr ZOUI from src/data_sources/rosreestr_zouit/gdal_source.xml 

In case 1 it's probably a config problem since the service has changed: Currently it points to http://api.atlas.mos.ru/arcgis/rest/services/Basemaps/egipdata/MapServer . It could be perhaps 'egipmap' which works like this http://api.atlas.mos.ru/arcgis/rest/services/Basemaps/egipmap/MapServer but not in QMS.

In case 2 it's pointing to this service http://maps.rosreestr.ru/arcgis/rest/services/Cadastre/ZOUIT/MapServer . This could be service provider problem since it returns an empty image.

**Comment by drnextgis** [comment]
I'm not sure this is a bug. Such sources shiuld works only with GDAL>=2.0.

**Comment by sfkeller** [comment]
Yes, GDAL works. But this russian AGS service which you deliver as core/contributed has changed from http://api.atlas.mos.ru/arcgis/rest/services/Basemaps/egipdata/MapServer . to perhaps this http://api.atlas.mos.ru/arcgis/rest/services/Basemaps/egipmap/MapServer ?

**Comment by drnextgis** [comment]
1. Confirm problem with **eAtlas EGIP**.
2. **Rosreestr ZOUIT** layer works (try to zoom in to Moscow):
![screenshot from 2016-06-01 13 57 27](https://cloud.githubusercontent.com/assets/866124/15700881/03bad952-2801-11e6-9906-39672a8f9cd0.png)
